docs/tutorials/working_with_the_ensemble.ipynb

```diff
@@ -19,7 +19,7 @@
   },
```

Comment:
<ol><li>Should we have this error message here?</li><li>Could we explain why we use <code>.copy()</code>?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<ol><li>No, we shouldn't have included the output in the PR, but this error seems to have been from running the cell once.</li><li>Here was just a simple way of creating a new frame, added a note</li></ol>